pylti/common.py

```diff
@@ -216,6 +217,7 @@ def post_message(consumers, lti_key, url, body):
     :param body: xml body
     :return: success
     """
+    # pylint: disable=unused-variable
```

Comment:
again, you should actually remove, or replace with an _ any variable that is unused